jekyll/_cci2/authorize-google-cloud-sdk.md

```diff
@@ -4,62 +4,66 @@ title: Authorize Google Cloud SDK
 categories: [deploying]
 redirect_from: /google-auth
-contentTags: 
+contentTags:
   platform:
   - Cloud
   - Server 3.x
   - Server 2.x
 ---
 
-This document explains how to install and authorize the [Google Cloud SDK](https://cloud.google.com/sdk/) in your primary container.
+This tutorial explains how to install and authorize the [Google Cloud SDK](https://cloud.google.com/sdk/) in your primary container.
 
-* TOC
-{:toc}
-
-## Overview
-{: #overview }
-{:.no_toc}
+## Introduction
+{: #introduction }
 
 The Google Cloud SDK is a powerful set of tools that can be used to access Google Cloud Platform (GCP) services like Google Compute Engine and Google Kubernetes Engine. On CircleCI, the Google Cloud SDK is recommended to deploy your application to GCP products.
 
 ## Prerequisites
 {: #prerequisites }
-{:.no_toc}
 
-- A CircleCI project.
-- A GCP project.
+- A CircleCI project
+- A GCP project
 
-### Installing the Google Cloud SDK
-{: #installing-the-google-cloud-sdk }
+## 1. Install the Google Cloud SDK
+{: #install-the-google-cloud-sdk }
 
-If Debian is an acceptable operating system for your primary container, consider using Google's base Docker image.
-You can find this image on DockerHub as [`google/cloud-sdk`](https://hub.docker.com/r/google/cloud-sdk/).
+If Debian is an acceptable operating system for your primary container, consider using Google's base Docker image. You can find this image on DockerHub as [`google/cloud-sdk`](https://hub.docker.com/r/google/cloud-sdk/).
 
 Otherwise, follow the [Google Cloud SDK installation instructions](https://cloud.google.com/sdk/) for your base image's operating system.
 
-### Creating and storing a service account
-{: #creating-and-storing-a-service-account }
+## 2. Create and store a service account
+{: #create-and-store-a-service-account }
 
 Before you can use any tools in the Google Cloud SDK, you must authorize `gcloud`. Google offers two types of authorization: user accounts and service accounts. Because you are installing the Cloud SDK on CircleCI, the service account is the appropriate choice.
 
-1. Create a service account by following Steps 1-3 of [Google's instructions](https://cloud.google.com/sdk/docs/authorizing#authorizing_with_a_service_account). Remember to download the JSON-formatted key file.
+### 1. Create a service account
+{: #create-a-service-account }
+
+Create a service account by following Steps 1-3 of [Google's instructions](https://cloud.google.com/sdk/docs/authorizing#authorizing_with_a_service_account). Remember to download the JSON-formatted key file.
+
+### 2. Add a key file
+{: #add-a-key-file }
+
+Add the key file to CircleCI as a [project environment variable](/docs/set-environment-variable/#set-an-environment-variable-in-a-project). In this example, the variable is named `GCLOUD_SERVICE_KEY`. Using this particular name is not required, but it will be used throughout the examples in this document.
+
+### 3. Add environment variables
+{: #add-env-vars }
```

Comment:
I wonder if these three steps are ok as an ordered list?

Reply:
It was an ordered list before and it looked quite messy on the page. Each bullet point is quite long, which is why I made it individual steps. I do think it looks better as individual steps. It's much easier to read.